README.md

```diff
@@ -27,6 +27,16 @@ The provided data comes from PHPs built-in function [`timezone_identifiers_list(
 If you want to use the dropdown field you can
 
 ```
+<?php
+class HomePage extends BaseHomePage {
+
+	private static $db = array(
```

Comment:
Thanks, I should have thought about this. It was really missing to understand this better